user_manual/Makefile

```diff
@@ -147,9 +147,9 @@ info:
 	@echo "makeinfo finished; the Info files are in $(BUILDDIR)/texinfo."
 
 gettext:
-	$(SPHINXBUILD) -b gettext $(I18NSPHINXOPTS) $(BUILDDIR)/locale
+	$(SPHINXBUILD) -b gettext $(I18NSPHINXOPTS) _catalog_templates
 	@echo
-	@echo "Build finished. The message catalogs are in $(BUILDDIR)/locale."
+	@echo "Build finished. The message catalogs are in ./_catalog_templates."
```

Comment:
Not fond of using this folder name, feels too technical. We use l10n everywhere on Nextcloud, I guess we can as well?

Reply:
I know, but basically, we need 2 folders:
 - 1 for source files (catalog templates)
 - 1 for translated files (message catalogs)

I got these terms from:
https://www.sphinx-doc.org/en/master/usage/advanced/intl.html

If you see for destination, I'm already using locale: `user_manual/locale/<lang>/LC_MESSAGES`

Or it could be `usermanual/locale/source` ? What do you think?

Just trying to think about my future self, I'll read again the doc, and the names will make total sense :)

Reply:
> I know, but basically, we need 2 folders:
> 
> * 1 for source files (catalog templates)
> * 1 for translated files (message catalogs)

We use a single one in vue. Works fine as the names are different :)

Reply:
https://github.com/nextcloud/nextcloud-vue/blob/master/l10n/messages.pot

Reply:
so we use locale like the default for sphinx.